### ISSUE TITLE
Unit spinners empty

### ISSUE THREAD

**Issue by quozl** [sugarlabs/convert#7]
Unit spinners on top far left and far right can be selected, but they don't show the current unit name.

![screenshot of _convert activity_](https://user-images.githubusercontent.com/2936481/38585632-e3fb0186-3d5d-11e8-90bf-8b0ffb8c694f.png)

**Comment by chimosky** [comment]
Addressed in #9 , made mistakes before committing so the rough commit history if you can rebase it'll be cool -- tried that and didn't work out well --.

**Comment by quozl** [comment]
There are so many rebase methods.

As you only have one pending change, the simplest rebase method for you should be to reset your index to the point of departure from https://github.com/sugarlabs/convert (which is 27dc884), commit your change again, cross-check against your remote https://github.com/chimosky/convert and force push.

Specific advice is always made harder by the different ways that people clone or name their remotes. Commands below assume that `origin` is the name of your `git remote` for https://github.com/chimosky/convert

```bash
git checkout master
git log  # check, index should be pointing at a60dcd7
git reset 27dc884
git log  # check, index will be pointing at 27dc884
git diff  # check, should look like https://github.com/sugarlabs/convert/pull/9/files
git add activity.py
git commit
git diff origin/master  # should be empty, proving you did it right
git push --force
```

Should any errors occur, stop to understand why.

Much of the above is checking so as to explain.  Once you get used to it, and gain confidence in the effects, it will reduce to a `git reset`, `git add`, `git commit`, and `git push`.

http://www.ndpsoftware.com/git-cheatsheet.html#loc=index; can be useful.

**Comment by chimosky** [comment]
Thanks

**Comment by quozl** [comment]
Fixed as of 7bb29e4f670b2dae1dfdd645fe0c7c6b83c75049.
